src/scaict_uwu/service_container.py

```
@@ -0,0 +1,91 @@
+        self._service_instantiators[name] = instantiator
+
+    def redefine_service(self, name: str, instantiator) -> None:
+        if not self.has_service(name):
+            sys.exit("NoSuchServiceException $name")
+
+        if name in self._services:
+            sys.exit("CannotReplaceActiveServiceException $name")
+
+        self._service_instantiators[name] = instantiator
+
+    def create_service(self, name: str):
```

Ruff (0.8.0)

Missing return type annotation for public function `create_service`

(ANN201)